### PR TITLE
SISRP-18106 - Updates University Requirements card to support Exempt status

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.berkeley.edu/browse/SISRP-18106

### CODE REVIEW

File: app/models/my_academics/requirements.rb [hunk 1]
```diff
@@ -13,13 +13,18 @@ def merge(data)
...
-                 when 'AMERICANINSTITUTIONS' then 'American Institutions'
-                 when 'AMERICANCULTURES' then 'American Cultures'
-                 else req_name
-               end
-        display_status = req_value.upcase == 'REQT SATISFIED' ? 'met' : ''
+          when 'SUBJECTA' then 'UC Entry Level Writing'
+          when 'AMERICANHISTORY' then 'American History'
+          when 'AMERICANINSTITUTIONS' then 'American Institutions'
+          when 'AMERICANCULTURES' then 'American Cultures'
+          else req_name
```
Comment (johncrossman) [inline app/models/my_academics/requirements.rb]: @redconfetti Should this be `req_name.upcase`?

**Comment by johncrossman** [comment]
I put one question, @redconfetti  Otherwise, looks good.

File: app/models/my_academics/requirements.rb (hunk 1 above)
Comment (redconfetti) [inline app/models/my_academics/requirements.rb]: @johncrossman Good question. No, I don't think so. This method is converting the uppercase representations into their lowercase equivalents. I could convert it to lowercase, but there isn't any way to properly space between the words in whatever value that might be.

The front-end code (for this card/widget) is only configured to respect these configured keys, so it wouldn't matter much anyway.

**Comment by johncrossman** [comment]
Thanks @redconfetti

**Pull request merged**